js/data-parsing/bitfinex.js

```diff
@@ -74,10 +73,10 @@ function openWebSocket() {
 
 // Export constructor that populates highbids and lowasks, returning another
 // object with exposed public functions
-module.exports = function(exchangeData) {
-  highbids = exchangeData.highbids;
-  lowasks = exchangeData.lowasks;
+module.exports = function(Exchs) {
+  highbids = Exchs.highbids;
+  lowasks = Exchs.lowasks;
   return {
-    openWebSocket: openWebSocket
+    openFeed: openWebSocket
```

Comment:
Nvm I see that it's because kraken's "feed" is an api call interval as opposed to a websocket. Can you mention this in a comment at each `openFeed` in each file? It may seem verbose, but it establishes something that might otherwise be confusing to a new dev on the project.